core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -15,8 +15,12 @@
 import alluxio.Configuration;
 import alluxio.Constants;
 import alluxio.PropertyKey;
+import alluxio.exception.ExceptionMessage;
```

Comment:
It would be ideal if the creator of `FileOutStream` did not need to differentiate between the two types and the logic was handled at the level of the `OutputStream` returned by the ufs.